lib/impure/re.nim

```diff
@@ -25,6 +25,8 @@
 ## .. include:: ../doc/regexprs.txt
 ##
 
+{.deprecated.}
```

Comment:
Having tried to use 'nre' in practice, I like to fix re's bugs instead of deprecating it.